src/app/gql/CLINICAL_SCHEMA_VERSION.ts

```diff
@@ -18,11 +18,15 @@
  */
 
 import { gql } from '@/__generated__/gql';
+import { apiName } from '@/lib/gql';
 
 const CLINICAL_SCHEMA_VERSION_QUERY = gql(`
   query ClinicalSchemaVersion {
     clinicalSubmissionSchemaVersion
   }
 `);
 
-export default CLINICAL_SCHEMA_VERSION_QUERY;
+export const ClinicalSchemaVersionQuery = {
+	gql: CLINICAL_SCHEMA_VERSION_QUERY,
+	api: apiName.clinical,
```

Comment:
OK so just confirming, going forward all queries will need to specify which API they point at?

Should we make Clinical the default? Could potentially reduce some extra lines of code

Reply:
Yes that is correct
We can add a default, but it's only an extra line of code. I don't mind being a little bit more explicit. Especially if it makes it clear

Reply:
While i don't hold a strong opinion, and do see the benefit of having a default value, intuitively I'm inclined towards advocating for having the single "Query" helper be agnostic, and then have two separate wrappers that each passes which endpoint you intend on calling. e.g. endpoint1QueryFn and endpoint2QueryFn, both based on baseQueryHelperFn.

Reply:
I do like wrapperA vs wrapperB pattern

My original thinking is that Gateway is only needed for one query (currently... could change)
So if majority of queries are all pointing to Clinical, you shouldn't have to specify that for every query
Why change all queries when only 1 needs to be different

Reply:
that's a fair point. I'm just trying to prevent the "you have to know this thing that is abstracted elsewhere" that is so on brand with our previous apps, which makes on-boarding so painful.

Reply:
Yeah exactly, I think wrapper is the best approach for that reason!